core/services/vagrant_service.rb

```diff
@@ -32,14 +58,12 @@ def self.up(provider, node, logger, path = Dir.pwd)
   end
 
   def self.package(node_name, box_name, logger, path = Dir.pwd)
-    if self.chek_distro_is_ubuntu_or_mint && !self.chek_vmlinuz_access_rights
-      return Result.error('Incorrect permissions for vmlinuz. Please run setup-dependencies')
-    end
+    return Result.error('Error in setting rights') unless self.set_access_rights_for_ubuntu_or_mint(logger)
 
     ShellCommands.run_command_in_dir(logger,
                                      "vagrant package #{node_name} --output #{box_name} --info info.json", path)
 
-    SUCCESS_RESULT
+    return SUCCESS_RESULT
```

Comment:
In Ruby the last value of the function is returned to the caller. No need to add extra return.